src/components/drawer/bl-drawer.stories.mdx

```diff
@@ -29,46 +32,42 @@ import { Meta, Canvas, ArgsTable, Story, Preview, Source } from '@storybook/addo
+  </p>
+  <p>
+    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+  </p>
+  <p>
+    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+  </p>
+  <p>
+    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
+  </p>
 `
 
 export const DrawerTemplate = (args) => html`
   <bl-drawer id=${ifDefined(args.id)}
              caption=${ifDefined(args.caption)}
              embed-url=${ifDefined(args.embedUrl)}
              external-link=${ifDefined(args.externalLink)}>
-  ${ifDefined(args.content)}
+    <div style="font: var(--bl-font-body-text-2)">
+      ${ifDefined(args.content)}
+    </div>
   </bl-drawer>
 `
 
 export const StoryTemplate = (args) => html`
   ${DrawerTemplate(args)}
-  <bl-button @click="${(event)=>openDialog(event,args)}">${ifDefined(args.buttonText)}</bl-button>
+  <bl-button @click="${(event)=>{document.getElementById(args.id).open=true}}">${ifDefined(args.buttonText)}</bl-button>
```

Comment:
Since I'm trying to understand if button is removed from DOM, I need to re-check it. Otherwise the reference of previous button (`closeBtn`) still exists and test fails.